publication/src/main/scala/hmda/publication/regulator/lar/RegulatorLarPublisher.scala

```diff
@@ -95,6 +96,7 @@ class RegulatorLarPublisher extends HmdaActor with LoanApplicationRegisterCassan
 
       val source = readData(fetchSize)
         .via(filterTestBanks)
+        .via(filterDynamicTestBanks)
```

Comment:
A little late to the party on this one but it feels strange to have the dynamic publishing occur in a file named `RegulatorLarPublisher`. Not looking for a change on this but it might be a nice, low priority, refactor down the line.